Assets/Scripts/GameScene/EnemiesGenerator.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+public class EnemiesGenerator : MonoBehaviour
+{
+    [SerializeField] private GameObject molePrefab;
+    [SerializeField] private int numOfMoles;
+    private List<GameObject> _moles = new List<GameObject>();
+    private HashSet<Vector3> _staticEnemies = new HashSet<Vector3>();
+    private bool boardDidLoad = false;
+    private Vector3[,] boardData;
```

Comment:
some names have `_[name]` and some don't, please pick one of them and stick with it...

Reply:
wrote about this in the previous PR. according to Rider IDE:
public should just be lowercase
[SerializeField] private should just be lowercase
private should start with '_'

It's a bit weird, will try to find the most common in the industry. Or we can just go according to regular acceptances?